run-ab-platform.sh

```diff
@@ -106,6 +108,9 @@ for argument in $@; do
     -x | --debug)
       set -o xtrace  # -x display every line before execution; enables PS4
       ;;
+    -b | --background)
+      dockerDetachedMode="-d"
+      ;;
```

Comment:
@cpdeethree can we have a flag or command to stop the instance? Because today people only will the `docker-compose down` anyway. Having this we could update the upgrading docs and everything use the script.

Reply:
@marcosmarxm I think any command to stop the instance would basically just wrap `docker compose down`. I feel like that is an acceptable command to use, since users should know that they are running docker containers via compose. `run-ab-platform.sh` is useful for starting the instance because it pulls the correct files/images, but it does not really add anything for stopping the instance.